udata_tabular_preview/__init__.py

```diff
@@ -1,8 +1,8 @@
 '''
 Tabular Preview
 
-A plugin to preview tabular files (CSV, Excel, ...)
+A plugin to preview tabular CSV files
```

Comment:
We want to revert to CSV & Excel right? It has been modified at different places in code. See https://github.com/etalab/udata-hydra/pull/92

Reply:
yes dev has been done before @Pierlou decided to preview excel file 😅